assets/js/blocks/product-categories/frontend.js

```diff
@@ -1,29 +1,31 @@
 /**
  * External dependencies
  */
-import { forEach } from 'lodash';
-import { render } from '@wordpress/element';
+import { render } from 'react-dom';
 
 /**
  * Internal dependencies
  */
 import Block from './block.js';
+import getCategories from './get-categories';
 
 const containers = document.querySelectorAll(
 	'.wp-block-woocommerce-product-categories'
 );
 
 if ( containers.length ) {
-	forEach( containers, ( el ) => {
+	containers.forEach( function( el ) {
```

Comment:
There seems to be an issue with the native NodeList `forEach()` and IE11:
![image](https://user-images.githubusercontent.com/3616980/61965701-22e57700-afd1-11e9-9ce3-67594a3aefe1.png)
Using Array's `forEach` should do the trick:
```suggestion
	Array.prototype.forEach.call( containers, ( el ) => {
```

For reference, I took the [work-around from MDN](https://developer.mozilla.org/en-US/docs/Web/API/NodeList#Example).